src/js/monkey.js

```diff
@@ -48,8 +49,13 @@ window.Monkey = module.exports = (function () {
       });
 
     if (options.letters) {
-      promise = promise.then(Monkey.letters._generateHtml(options.letters, options.lang))
-        .then(Monkey.letters._init(this.$events));
+      promise = promise.then(Monkey.letters._generateHtml(
```

Comment:
If it a function has enough arguments that they need to be wrapped, it should accept an object. Isn't JSCS checking for this?